wallet/src/de/schildbach/wallet/ui/explore/ExploreActivity.kt

```diff
@@ -1,98 +0,0 @@
-/*
- * Copyright 2021 Dash Core Group.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-package de.schildbach.wallet.ui.explore
-
-import android.content.Intent
-import android.net.Uri
-import android.os.Bundle
-import androidx.activity.viewModels
-import androidx.lifecycle.lifecycleScope
-import dagger.hilt.android.AndroidEntryPoint
-import de.schildbach.wallet.data.BlockchainStateDao
-import de.schildbach.wallet.ui.BaseMenuActivity
-import de.schildbach.wallet.ui.payments.PaymentsActivity
-import de.schildbach.wallet.ui.staking.StakingActivity
-import de.schildbach.wallet_test.R
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.FlowPreview
-import kotlinx.coroutines.launch
-import org.dash.wallet.common.ui.dialogs.AdaptiveDialog
-import org.dash.wallet.features.exploredash.ui.ExploreViewModel
-import org.dash.wallet.features.exploredash.ui.NavigationRequest
-import javax.inject.Inject
-
-@AndroidEntryPoint
-@FlowPreview
-@ExperimentalCoroutinesApi
-class ExploreActivity : BaseMenuActivity() {
```

Comment:
Not needed anymore. Explore will be hosted in the `WalletActivity` and its nav graph included in the main nav graph.

The same can be done for other feature modules in the future; this is a good first step towards a single activity architecture.